package.json

```diff
@@ -12,18 +12,25 @@
 		"install:packages": "cd packages && yarn install",
 		"start": "yarn install && yarn dev",
 		"dev": "node ./scripts/run.js",
+		"watch": "yarn dev",
 		"restart:meteor": "node ./scripts/meteor-force-restart.js",
+		"build": "yarn install && yarn build:packages",
```

Comment:
`build` should just build the project. It shouldn't also install it.

Reply:
I've updated the branch.